src/api/auth/auth.controller.ts

```diff
@@ -4,18 +4,27 @@
 import { AuthService } from './auth.service';
 import { LoginDto } from './dto/login.dto';
 
+
 @ApiTags('Auth')
 @Controller('auth')
 export class AuthController {
   constructor(private readonly authService: AuthService) {}
 
   @Post('sign-up')
-  async register(@Body() register: CreateUserDto) {
+  async register(@Body() register: CreateUserDto, hre: any) {
+    
+    let var1;
+    if (register.username === '') {
+      var1 = 'Username is required';
+    } else {
+      var1 = ''
+    }
```

Comment:
🚫 **[eslint]** <[unicorn/prefer-ternary](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v56.0.0/docs/rules/prefer-ternary.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>This `if` statement can be replaced by a ternary expression.
```suggestion
    var1 = register.username === '' ? 'Username is required' : '';
```

<!-- __reviewdog__:ChA5YWY4MWUyMzA2Y2Q5Yzg1EgZlc2xpbnQ= -->